global.json

```diff
@@ -22,9 +22,9 @@
     }
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.24453.1",
-    "Microsoft.DotNet.Helix.Sdk": "9.0.0-beta.24453.1",
-    "Microsoft.DotNet.SharedFramework.Sdk": "9.0.0-beta.24453.1",
+    "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.24459.6",
```

Comment:
Since we are touching this file, can we update the SDK to use the recently released RC1 too?

Reply:
Isn't that something that arcade drives? They run a little behind the latest version. There can be issues with updating to a newer one which arcade would fix on it's side before updating everyone.

Reply:
Not necessarily. Arcade has a minimum required version, which will push to consuming repos if not met, but repos are allowed to (and even in some cases encouraged) go beyond this version and use a newer one. For instance, aspnetcore typically updates their SDK weekly.